functional/debug_console/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SCRIPT_PATH=$(dirname "$0")
+
+source "${SCRIPT_PATH}/../../lib/common.bash"
+
+extract_kata_env
+if [ -z "${INITRD_PATH}" ]; then
+	info "Skip: initrd path is empty"
+	exit 0
+fi
+
+
+info "Enable debug console"
+sudo cp "${RUNTIME_CONFIG_PATH}" "${RUNTIME_CONFIG_PATH}.old"
+
+info "Disable debug"
+sudo sed -i \
```

Comment:
I just saw we use `crudini` elsewhere to do conf file modding I think - should we try and standardize? /cc @chavafg

Reply:
yeah, I think it would be good to have a standard, I like `crudini` as it is more simpler to use (and clearer) and works very well for the `toml` file that we use for the kata config.